src/main/java/wooteco/subway/admin/dto/StationResponse.java

```diff
@@ -22,6 +25,12 @@ public StationResponse(Long id, String name, LocalDateTime createdAt) {
         this.createdAt = createdAt;
     }
 
+    public static List<StationResponse> listOf(List<Station> stations) {
+        return stations.stream()
+                .map(it -> StationResponse.of(it))
```

Comment:
```suggestion
                .map(StationResponse::of)
```

메서드 레퍼런스를 사용하면 가독성이 상승합니다.